content/docs/python/tracing.md

```diff
@@ -20,30 +20,31 @@ From there, you should be able to use opentelemetry as per the following:
 ```python
 from opentelemetry import trace
 from opentelemetry.context import Context
-from opentelemetry.sdk.trace import Tracer
+from opentelemetry.sdk.trace import TracerSource
```

Comment:
TracerSource has been renamed to TracerProvider - [opentelemetry-python/441](https://github.com/open-telemetry/opentelemetry-python/pull/441)

```suggestion
from opentelemetry.sdk.trace import TracerProvider
```